### DISCUSSION TITLE
[Prettier Plugin] Support for CVA

### DISCUSSION THREAD

**Discussion by itsjxck** [tailwindlabs/tailwindcss#9686] [Ideas]
Support for [Component Variance Authority (CVA)](https://github.com/joe-bell/cva) in the sorter would be super useful.

**Comment by alexcarpenter** [comment]
Yes, this would be super nice. CVA is really helpful for the adoption of Tailwind in our work.

**Comment by dcunited001** [comment]
damit

**Comment by luiferg** [comment]
Hi, well, you can have prettier for cva using the following in prettier config file

```
plugins: [
 'prettier-plugin-tailwindcss',
],
tailwindFunctions: ['cva'],
```

This saved me
